e2e/cypress/integration/account_settings/sidebar/channel_switcher_spec.js

```diff
@@ -28,7 +28,7 @@ describe('Account Settings > Sidebar > Channel Switcher', () => {
             const numberOfChannels = 14;
             Cypress._.forEach(Array(numberOfChannels), (_, index) => {
                 cy.apiCreateChannel(teamId, 'channel-switcher', `Channel Switcher ${index.toString()}`).then((response) => {
-                    if (index === numberOfChannels - 1) {
+                    if (index === 0) {
```

Comment:
Test is failing since "Channel Switcher 14" is out of view.  There's `scrollIntoView` but the position seemed to be not reliable and sometimes being covered by the floating "More Unreads" button.  Instead of using Channel 14, just use Channel 0 as test channel.